packages/core/content-type-builder/admin/src/components/FormModal/utils/getCustomFieldDefaultOptions.js

```diff
@@ -0,0 +1,16 @@
+const getCustomFieldDefaultOptions = (options, optionDefaults = []) => {
+  options.forEach((option) => {
+    if (option.items) {
+      getCustomFieldDefaultOptions(option.items, optionDefaults);
+    }
+
+    if ('defaultValue' in option) {
```

Comment:
I'm just curious why you chose defaultValue and not default for the API since I believe at first we explored default

Reply:
I'm a bit on the fence. `default` doesn't seem like the most explicit name since it doesn't specify what it is a default for. 

The `default` naming convention is already established on the root level options for the default value to be set in the content manager (I find the name a bit vague for that use case as well). We would be consistent there but would it also be confusing to have something like this where you are setting the default for the default?

```
{
  name: 'default',
  type: 'text',
  default: 'someval'
}
```

Reply:
Sounds good to me then! I wasn't saying we should change it, just wanted to understand your choice